51-android.rules

```diff
@@ -41,6 +41,15 @@ ATTR{idVendor}=="10d6", ATTR{idProduct}=="0c02", GOTO="adb"
 #   S5
 ATTR{idVendor}=="0a5c", ATTR{idProduct}=="e681", GOTO="adb"
 
+# Allwinner Technology
+ATTR{idVendor}!="1f3a", GOTO="not_Allwinner"
+#   Prestigio PER3464B ebook reader (Mass storage mode)
+ATTR(idProduct)=="1000", GOTO="adbmtp"
```

Comment:
Hi and thanks for your contribution. please use curly brackets

Reply:
Done